src/dispatch/messaging.py

```diff
@@ -280,7 +280,7 @@ class MessageType(str, Enum):
 INCIDENT_CONFERENCE = {
     "title": "Incident Conference",
     "title_link": "{{conference_weblink}}",
-    "text": INCIDENT_CONFERENCE_DESCRIPTION,
+    "text": INCIDENT_CONFERENCE_DESCRIPTION + " Password: {{conference_challenge}}",
```

Comment:
Can you add the password in the `INCIDENT_CONFERENCE_DESCRIPTION`?

Reply:
Sure